packages/@webex/plugin-meetings/src/meeting/index.ts

```diff
@@ -10,6 +10,7 @@ import {
   ClientEventLeaveReason,
   CallDiagnosticUtils,
   CALL_DIAGNOSTIC_CONFIG,
+  RtcMetrics,
```

Comment:
The param change [here](https://github.com/webex/webex-js-sdk/blob/b6fd626956725c6047eeca9827724eb847cdc911/packages/%40webex/internal-plugin-metrics/src/rtcMetrics/index.ts#L50) is not handled in this file [here](https://github.com/webex/webex-js-sdk/blob/b6fd626956725c6047eeca9827724eb847cdc911/packages/%40webex/plugin-meetings/src/meeting/index.ts#L6315)